xbmc/LangInfo.cpp

```diff
@@ -541,6 +570,45 @@ bool CLangInfo::Load(const std::string& strLanguage)
   }
   g_charsetConverter.reinitCharsetsFromSettings();
 
+  // CLog::Log(LOGDEBUG, StringUtils::Format("CLangInfo::Load '{}' '{}'", strLanguage, strFileName));
+  // CLog::Log(LOGDEBUG, StringUtils::Format("CLangInfo::Load m_languageCodeGeneral '{}'",
+  //                                         m_languageCodeGeneral));
+
+  // CLog::Log(LOGDEBUG, StringUtils::Format("CLangInfo::Load m_strLanguageConfigCode '{}'",
+  //                                         m_strLanguageConfigCode));
+  // CLog::Log(LOGDEBUG,
+  //           StringUtils::Format("CLangInfo::Load m_strLanguageISO6391 '{}'", m_strLanguageISO6391));
+  // CLog::Log(LOGDEBUG,
+  //           StringUtils::Format("CLangInfo::Load m_strLanguageISO6392 '{}'", m_strLanguageISO6392));
+  // CLog::Log(LOGDEBUG, StringUtils::Format("CLangInfo::Load m_strlanguageISOEnglishName '{}'",
+  //                                         m_strlanguageISOEnglishName));
+
+  // CLog::Log(LOGDEBUG, StringUtils::Format("CLangInfo::Load GetLanguageConfigCode() '{}'",
+  //                                         GetLanguageConfigCode()));
+  // CLog::Log(LOGDEBUG,
+  //           StringUtils::Format("CLangInfo::Load GetLanguageISO6391() '{}'", GetLanguageISO6391()));
+  // CLog::Log(LOGDEBUG,
+  //           StringUtils::Format("CLangInfo::Load GetLanguageISO6392() '{}'", GetLanguageISO6392()));
+  // CLog::Log(LOGDEBUG, StringUtils::Format("CLangInfo::Load GetLanguageISOEnglishName() '{}'",
+  //                                         GetLanguageISOEnglishName()));
+
+  // CLog::Log(LOGDEBUG,
+  //           StringUtils::Format("CLangInfo::Load m_defaultRegion.m_strLangLocaleName '{}'",
+  //                              m_defaultRegion.m_strLangLocaleName));
+  // CLog::Log(LOGDEBUG,
+  //           StringUtils::Format("CLangInfo::Load m_defaultRegion.m_strLangLocaleCodeTwoChar '{}'",
+  //                              m_defaultRegion.m_strLangLocaleCodeTwoChar));
+
+  // CLog::Log(LOGDEBUG,
+  //           StringUtils::Format("CLangInfo::Load GetCurrentRegion() '{}'", GetCurrentRegion()));
+  // CLog::Log(LOGDEBUG, StringUtils::Format("CLangInfo::Load GetCurrentRegionISO31661Alpha2() '{}'",
+  //                                        GetCurrentRegionISO31661Alpha2()));
+  // CLog::Log(LOGDEBUG, StringUtils::Format("CLangInfo::Load GetCurrentRegionISO31661Alpha3() '{}'",
+  //                                         GetCurrentRegionISO31661Alpha3()));
+  // CLog::Log(LOGDEBUG,
+  //           StringUtils::Format("CLangInfo::Load GetCurrentRegionISO31661EnglishName() '{}'",
+  //                               GetCurrentRegionISO31661EnglishName()));
+
```

Comment:
Remove all this if its not intended to be enabled. Same with the other block

Reply:
Done